swatch-contracts/ct/java/tests/ContractsComponentTest.java

```diff
@@ -227,4 +232,100 @@ private void verifyNextEventDateIsTomorrowAtEndOfDay(OffsetDateTime nextEventDat
     assertThat("next_event_date minute should be 59", nextEventDate.getMinute(), equalTo(59));
     assertThat("next_event_date second should be 59", nextEventDate.getSecond(), equalTo(59));
   }
+
+  @Test
+  @Tag("contract")
+  @Tag("capacity-validation")
+  void shouldValidateSumOfAllSocketsForVirtWhoSkus() {
```

Comment:
Please, move this test along with the rest of the tests (next to the test "shouldUpdateSubscriptionTableWhenContractIsTerminated"). So, all the tests are written together.